src/plugins/discover/public/application/main/components/top_nav/open_alerts_popover.tsx

```diff
@@ -98,7 +98,7 @@ export function AlertsPopover({
 
     return triggersActionsUi?.getAddRuleFlyout({
       metadata: discoverMetadata,
-      consumer: STACK_ALERTS_FEATURE_ID,
+      consumer: 'alerts',
```

Comment:
The `ALERTS_FEATURE_ID` is exported from the x-pack alerts plugin, which we're not allowed to import from `/src`

Reply:
@Zacqary What about serverless? As part of this [PR](https://github.com/elastic/kibana/commit/a35f91e3a5bfa9cd3f1aa03fd4b4a30911c8e014#diff-f05249ca8e6ef2d4835afd6fb32a0985927ec7ed73dd600c2b3aceca9b0a8873R109) the `observability` feature was added and I added a [test ](https://github.com/elastic/kibana/commit/a35f91e3a5bfa9cd3f1aa03fd4b4a30911c8e014#diff-f05249ca8e6ef2d4835afd6fb32a0985927ec7ed73dd600c2b3aceca9b0a8873R109)for es-query that expects consumer is observability 

I will test your PR locally later on to verify that things work as expected on serverless as well. Did you test it there?

Reply:
I read the [Slack discussion](https://elastic.slack.com/archives/C023GDA0WMP/p1698922856755039), and it is mentioned that serverless should be fine. This scope dropdown is added only for stateful.